### ISSUE TITLE
Desync on animation in DS

### ISSUE THREAD

**Issue by nukeguard** [sstixrud/WeaponCore#240]
## Is this a multiplayer or singleplayer bug?

MP

## Describe the bug:

Animation plays for client activating animation, but does not play for another client observing action.

## Steps To Reproduce:

Have a turret mod with on / off animations in DS 
at least two players connected to DS
place an animated turret
toggle on/off for animations

## Expected behavior:
both clients observing animation should see it on DS

### User Error Report (if Weapon-Specific) and log files
*WeaponCore lets you generate an error report for a specific weapon, while ingame, hold Ctrl+R and aim the red line at the Weapon that is erroring, and an error report will be generated in WeaponCores log cache: `Users\UserName\AppData\Roaming\SpaceEngineers\Storage\1918681825.sbm_WeaponCore`

Zip up that folder and link it here:

**Comment by nukeguard** [comment]
These are amazing. I want to use these on everything.

Small bug report:

On a Dedicated Server. There seems to be a de-sync with the retraction.

1. When building the turret, once complete, both people see it exposed, or "up"

2. When the person who built it turns it off, they see it retract as intended, but the other person does not. (they stay extended for that person)

*Other person hears audio for retraction*

3. When the person who built them turns them on again, they extend as expected, but for the other person, they "go up" another block.

https://i.imgur.com/xsp9eOD.png


I'm on a dedicated server with one other person. He built the first set, when I saw the bug.
I grided them down and rebuilt them and he saw the behavior I described.

I am a software developer for a living, so while my experience with SE mods is zero, I am willing to provide any kind of testing or feedback that may help. And wouldn't mind learning a bit in the process.
